js/components/Table/table.scss

```diff
@@ -2,8 +2,7 @@
  * Table
  */
 @import '~@dosomething/forge/scss/toolkit';
-@import './sass/app';
-
+@import '../../../sass/toolbox.scss';
 
```

Comment:
@DFurnes I'm still getting 
```
ERROR in ./js/components/Table/table.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Can't resolve '../site/fonts/leaguegothic-regular.woff' in '/Users/chloelee/DoSomething.org/domino/js/components/Table'
```
is this because I'm also importing the `toolbox.scss` in `app.scss`? If I don't include it in `app.scss`, then it can't find the variables from `toolbox.scss`. I tried to just import `fonts` and add things manually but that still gave me the above error?

Reply:
The fonts should be imported in `app.scss`! The `toolbox.scss` file should just have variables, functions, & mix-ins.

Reply:
OH yes you mentioned that before and I just didn't realize I was doing it wrong - thanks @DFurnes !